inspirehep/config.py

```diff
@@ -55,6 +55,7 @@
 FEATURE_FLAG_ENABLE_MERGER = False
 FEATURE_FLAG_ENABLE_UPDATE_TO_LEGACY = False
 """This feature flag will prevent to send a ``replace`` update to legacy."""
```

Comment:
Does this comment refer to the next new line `FEATURE_FLAG_ENABLE_SENDING_REFERENCES_TO_LEGACY = False`? Can you make it more clear? And format it as a regular comment starting with `#`?

I'm sorry for not having seen it earlier.

Reply:
this is module level variable documented inline ([example](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)), we might need to define the style we're gonna use in the future